crates/swc_ecma_minifier/tests/terser/compress/harmony/class_name_can_be_preserved/output.js

```diff
@@ -1,4 +1,6 @@
 function x() {
-    (class Baz {});
-    class Foo {}
+    (class a {
```

Comment:
This change is due to we didn't implement the option of terser to keep class names.